content/spin/ai-sentiment-analysis-api-tutorial.md

```diff
@@ -137,6 +147,22 @@ HTTP path: /api/...
 
 {{ blockEnd }}
 
+{{ startTab "TinyGo" }}
+
+> Note: please add `/api/...` when prompted for the path; this provides us with an API endpoint to query the sentiment analysis component.
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin new http-go 
+Enter a name for your new application: sentiment-analysis
+Description: A sentiment analysis API that demonstrates using LLM inferencing and KV stores together
+HTTP base: /
```

Comment:
This is no longer prompted for in the v2 templates

Reply:
And done.